js/index.js

```diff
@@ -76,7 +83,7 @@ fetch(apiUrl)
                     </div>
                     <ul class="list-group list-group-flush">
                     <li class="list-group-item font-weight-bold">Price: <span class="text-danger h4">${element.preis} Euro</span></spand></li>
-                    <li class="list-group-item text-muted">Basic Price: <span class="text-danger">${element.basicPrice}</span></spand></li>
+                    <li class="list-group-item list-price text-muted" id="basic-price">Basic Price: <span class="text-danger">${element.basicPrice}</span></spand></li>
```

Comment:
here you will have the same exact id for each and every element in the data.docs but `id` attribute should be always unique